src/main/java/algorithms/leetcode/easy/BackSpaceStringCompare.java

```diff
@@ -0,0 +1,21 @@
+package algorithms.leetcode.easy;
+
```

Comment:
Where are the details about this problem?

Reply:
added details